gufe/visualization/mapping_visualization.py

```diff
@@ -153,15 +153,26 @@ def _draw_molecules(
 
     # squash to 2D
     copies = [Chem.Mol(mol) for mol in mols]
-    for mol in copies:
-        AllChem.Compute2DCoords(mol)
-
-    # mol alignments if atom_mapping present
-    for (i, j), atomMap in atom_mapping.items():
-        AllChem.AlignMol(
-            copies[j], copies[i],
-            atomMap=[(k, v) for v, k in atomMap.items()]
-        )
+    # create layout of first molecule
+    # TODO:
```

Comment:
Please raise an issue for this current problem and this todo.

I suspect someone's going to stumble upon this with an old version of gufe and we'll forget this ever happened.